README.md

```diff
@@ -34,115 +63,157 @@ You can also tab complete remote names and branch names e.g.: `git pull or<tab>
 
 Note on performance: Displaying file status in the git prompt for a very large repo can be prohibitively slow.
 Rather than turn off file status entirely (`$GitPromptSettings.EnableFileStatus = $false`), you can disable it on a repo-by-repo basis by adding individual repository paths to `$GitPromptSettings.RepositoriesInWhichToDisableFileStatus`.
 
 ## Installation
+
 ### Prerequisites
+
 Before installing posh-git make sure the following prerequisites have been met.
 
-1. PowerShell 3.0 or higher. Check your PowerShell version by executing `$PSVersionTable.PSVersion`.
+1. Windows PowerShell 5.x or PowerShell Core 6.0.
+   Get PowerShell Core 6.0 for Windows, Linux or macOS from [here][pscore-install].
+   You can check your PowerShell version by executing `$PSVersionTable.PSVersion`.
 
-2. Script execution policy must be set to either `RemoteSigned` or `Unrestricted`.
+1. On Windows, script execution policy must be set to either `RemoteSigned` or `Unrestricted`.
    Check the script execution policy setting by executing `Get-ExecutionPolicy`.
    If the policy is not set to one of the two required values, run PowerShell as Administrator and execute `Set-ExecutionPolicy RemoteSigned -Scope CurrentUser -Confirm`.
 
-3. Git must be installed and available via the PATH environment variable.
+1. Git must be installed and available via the PATH environment variable.
    Check that `git` is accessible from PowerShell by executing `git --version` from PowerShell.
    If `git` is not recognized as the name of a command verify that you have Git installed.
    If not, install Git from [https://git-scm.com](https://git-scm.com).
    If you have Git installed, make sure the path to git.exe is in your PATH environment variable.
 
-### Installing posh-git via PowerShellGet
-If you are on PowerShell version 5 or higher, execute the command below to install from the [PowerShell Gallery](https://www.powershellgallery.com/):
+### Installing posh-git
+
+posh-git is available on the [PowerShell Gallery][psgallery] and can be installed using the PowerShellGet module.
+Start either Windows PowerShell 5.x or PowerShell Core 6.x (`pwsh`) and execute the following command:
 
 ```powershell
-PowerShellGet\Install-Module posh-git -Scope CurrentUser
+Install-Module posh-git -Scope CurrentUser -Force
 ```
-You may be asked if you trust packages coming from the PowerShell Gallery. Answer yes to allow installation of this module to proceed.
 
-If you are on PowerShell version 3 or 4, you will need to install the [Package Management Preview for PowerShell 3 & 4](https://www.microsoft.com/en-us/download/details.aspx?id=51451) in order to run the command above.
+- If you are asked to trust packages coming from the PowerShell Gallery.
```

Comment:
1. Could PowerShell Gallery troubleshooting tips be punted to a wiki page (here or a more canonical source)?
2. If we keep them here, this rephrased sentence should be joined to the following line with a comma.

Reply:
Absolutely could punt.  Wasn't sure about keeping this so I can be trivially talked into removing it.  :-)

Reply:
Since these are common hang-ups, and posh-git is often the only PS module a dev may have installed, I want to make sure guidance is available. Open to thoughts on where that would best live.

Reply:
Since these are common hang-ups, and posh-git is often the only PS module a dev may have installed, I want to make sure guidance is available. Open to thoughts on where that would best live.

Reply:
The trust issue is pretty common so I recommend we keep the troubleshooting help here.